### PR TITLE
Clarify relation to RFC 4949 terms

### PR DESCRIPTION
**Pull request by laurencelundblade**
RFC 4949 system component == EAT entity
RFC 4949 system entity ~= EAT entity
RFC 4949 entity ~= EAT entity

Note that RFC 4949 system entity can be an RFC 4949 subsystem which in turn can contain an RFC 4949 system component. So the equivalences above work.

### CODE REVIEW

**Review by quic-jodonogh — APPROVED** [review]

File: draft-ietf-rats-eat.md [hunk 1]
```diff
@@ -241,9 +241,13 @@ The document uses the term "entity" to refer to the target of an EAT. Many of th
 Correspondingly, the EAT format allows claims to be organized using mechanisms like submodules and nested EATs (see {{submods}}).
 The entity to which a claim applies is the submodule in which it appears, or to the top-level entity if it doesn't appear in a submodule.
 
-An entity also corresponds to a "system component", as defined in the Internet Security Glossary {{RFC4949}}.
+An entity is the same as a "system component", as defined in the Internet Security Glossary {{RFC4949}}.
+
+Note however that the {{RFC4949}} definitions of the terms "entity" and "system entity" are not the same as the definition of entity here.
```
Comment (nedmsmith) [inline draft-ietf-rats-eat.md:246]:
```suggestion
Note that {{RFC4949}} defines  "entity" and "system entity" as synonyms, and may be a person or organization in addition to being a system component.
```

File: draft-ietf-rats-eat.md [hunk 2]
```diff
@@ -241,9 +241,13 @@ The document uses the term "entity" to refer to the target of an EAT. Many of th
...
 That glossary also defines "entity" and "system entity" as something that may be a person or organization as well as a system component.
```
Comment (nedmsmith) [inline draft-ietf-rats-eat.md:247]:
```suggestion

```

File: draft-ietf-rats-eat.md [hunk 3]
```diff
@@ -241,9 +241,13 @@ The document uses the term "entity" to refer to the target of an EAT. Many of th
...
-In the EAT context, "entity" never refers to a person or organization. The hardware and software that implement a server or service used by a web site may be an entity, but the organization that runs the web site is not an entity nor is the web site itself. An entity is an implementation in hardware, software or both.
+In the EAT context, "entity" never refers to a person or organization.
+The hardware and software that implement a server or service used by a web site may be an entity in the EAT sense, but the organization that runs the web site is not an entity nor is the web site itself.
```
Comment (nedmsmith) [inline draft-ietf-rats-eat.md:249]:
```suggestion
The hardware and software that implement a web site server or service may be an entity in the EAT sense, but the organization that operates, maintains or hosts the web site is not an entity.
```

**Comment by laurencelundblade** [comment]
@nedsmith and @carl-wallace -- this is a compromise between various proposals you made relative to draft-19. Note that the diff of this PR is not the same as against draft-19 because Carl's proposal already got merged.

**Review by carl-wallace — APPROVED** [review]

**Pull request merged**